test/utils/test_utils.py

```diff
@@ -152,7 +154,7 @@ def elaborate(self, platform):
         return m
 
 
-@pytest.mark.parametrize("size", range(1, 7))
+@pytest.mark.parametrize("size", range(1, 128))
```

Comment:
Notes from discussion:

```suggestion
@pytest.mark.parametrize("size", [1,2,3,4,5,6,7,8,9,63,64,65,97,98,127,128])
```
(less tests sample)